client/js/gdpr/table.js

```diff
@@ -7,6 +7,19 @@ export class GdprTable {
    */
   constructor(chart, histories, dates) {
     this.table = document.createElement('table');
+    this.table.classList.add('gdpr-table');
+
+    const thead = document.createElement('thead');
+    thead.innerHTML = 
+      '<th class="gdpr-table-header gdpr-table-header-rank">Rank</th>' +
+      '<th class="gdpr-table-header gdpr-table-header-track">Track</th>';
+
+    const tbody = document.createElement('tbody');
+    tbody.classList.add('gdpr-table-body');
+
+    this.table.append(thead, tbody);
```

Comment:
Move these below where you initialize your class variables. In general, it's easier to read if the majority of your declarations come first, then the bulk of the code happens afterwards.